cf/read_write/um/umread.py

```diff
@@ -1203,12 +1202,37 @@ def atmosphere_hybrid_height_coordinate(self, axiscode):
             field, ac, axes=[_axis["z"]], copy=False
         )
 
-        # atmosphere_hybrid_height_coordinate dimension coordinate
-        TOA_height = bounds1.max()
-        if TOA_height <= 0:
-            TOA_height = self.height_at_top_of_model
+        # Height at top of atmosphere
+        TOA_height = self.height_at_top_of_model
```

Comment:
Not specifically related to your changes, though in the spirit of tidying as per this PR, can we convert `TOA_height` to `toa_height` or some other similar fully non-capitalised name, because it is somewhat confusing and off the standard Python naming style to have the initial letters capitalised like an enumeration or constant name, then switching to all lower-case?

Reply:
Good idea. Fixed in https://github.com/NCAS-CMS/cf-python/pull/472/commits/df58afbcec71a6554ed40ed849d0bb9c9a52a7fa